src/renderer/components/publication/publicationInfos/PublicationInfo.tsx

```diff
@@ -160,6 +170,7 @@ export class PublicationInfo extends React.Component<Props, State> {
                         <span>{__("catalog.lang")}</span> { __(`languages.${formatedLanguages}`) } <br/>
                         <span>{__("catalog.id")}</span> { publication.workIdentifier } <br/>
                     </p>
+                    <button onClick={() => this.props.getPublicationFromId()}>lolilol</button>
```

Comment:
LOL, is that really meant to be here?? :)

Reply:
Oups, I forgot to remove it, sorry